src/app/shared/components/menu/main-menu/main-menu.component.ts

```
@@ -58,7 +48,8 @@
   readonly brandingFeatureConfig = brandingFeatureConfig;
 
   constructor(
-    private router: Router
+    private router: Router,
+    private integrationsService: IntegrationsService,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)